spec/rubocop/cop/rails/where_range_spec.rb

```diff
@@ -0,0 +1,200 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::WhereRange, :config do
+  context 'Ruby <= 2.5', :ruby25 do
+    it 'does not register an offense when using anonymous `>=`' do
+      expect_no_offenses(<<~RUBY)
+        Model.where('column >= ?', value)
+      RUBY
+    end
+
+    it 'does not register an offense when using anonymous `<=`' do
+      expect_no_offenses(<<~RUBY)
+        Model.where('column <= ?', value)
+      RUBY
+    end
+  end
+
+  context 'Rails 5.1', :rails51 do
+    it 'does not register an offense when using anonymous `>=`' do
+      expect_no_offenses(<<~RUBY)
+        Model.where('column >= ?', value)
+      RUBY
+    end
+  end
+
+  context 'Rails 6.0', :rails60 do
+    context 'Ruby >= 2.6', :ruby26 do
+      it 'registers an offense and corrects when using anonymous `>=`' do
+        expect_offense(<<~RUBY)
+          Model.where('column >= ?', value)
+                ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Use `where(column: value..)` instead of manually constructing SQL.
+        RUBY
+
+        expect_correction(<<~RUBY)
+          Model.where(column: value..)
+        RUBY
+      end
+
+      it 'registers an offense and corrects when using named `>=`' do
+        expect_offense(<<~RUBY)
+          Model.where('column >= :min', min: value)
+                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Use `where(column: value..)` instead of manually constructing SQL.
+        RUBY
+
+        expect_correction(<<~RUBY)
+          Model.where(column: value..)
+        RUBY
+      end
+
+      it 'registers an offense and corrects when using anonymous `>=` with an explicit table' do
+        expect_offense(<<~RUBY)
+          Model.where('table.column >= ?', value)
+                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Use `where(table: { column: value.. })` instead of manually constructing SQL.
+        RUBY
+
+        expect_correction(<<~RUBY)
+          Model.where(table: { column: value.. })
+        RUBY
+      end
+
+      it 'does not register an offense when using anonymous `>`' do
+        expect_no_offenses(<<~RUBY)
+          Model.where('column > ?', value)
+        RUBY
+      end
+
+      it 'does not register an offense when using anonymous `<=`' do
+        expect_no_offenses(<<~RUBY)
+          Model.where('column <= ?', value)
+        RUBY
+      end
+
+      it 'does not register an offense when using anonymous `<`' do
+        expect_no_offenses(<<~RUBY)
+          Model.where('column < ?', value)
+        RUBY
+      end
```

Comment:
Prism matrix is failure. Can you use `unsupported_on: :prism`?
https://github.com/rubocop/rubocop-rails/actions/runs/8860610072/job/24331577196?pr=1272

```suggestion
      context 'TargetRubyVersion <= 2.6', :ruby26, unsupported_on: :prism do
        it 'does not register an offense when using anonymous `<=`' do
          expect_no_offenses(<<~RUBY)
            Model.where('column <= ?', value)
          RUBY
        end

        it 'does not register an offense when using anonymous `<`' do
          expect_no_offenses(<<~RUBY)
            Model.where('column < ?', value)
          RUBY
        end
      end
```